test/e2e/cross_os/test_job_submissions.py

```diff
@@ -430,3 +439,83 @@ def test_worker_streams_logs_to_cloudwatch(
         )
 
         assert len(worker_logs["events"]) > 0
+
+    def test_worker_uses_job_attachment_configuration(
```

Comment:
This is moved from the linux test file, so that it will run on both linux and windows